docker/requirements/Checker.php

```
@@ -0,0 +1,639 @@
+        return $this->compareByteSize($memoryLimit, $minimumLimit, '>=');
+    }
+
+    /**
+     * Check if command exists in PATH
+     */
+    public function checkCommandExists($command)
+    {
+        $locator = (PHP_OS_FAMILY === 'Windows') ? 'where' : 'command -v';
+        $arg = escapeshellarg($command);
+        $code = 1;
+        exec("$locator $arg > /dev/null 2>&1", $void, $code);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$void'. (Unused Code Rules)

(UnusedLocalVariable)